@atykhyy one test failing:
```
--- FAIL: Test/PolicyAPITestSuite#01/TestToServicesSanitize (0.00s)

    rule_validation_test.go:630: 

        ... value *errors.errorString = &errors.errorString{s:"Combining ToServices and ToPorts is not supported yet"} ("Combining ToServices and ToPorts is not supported yet")
```

--- a/Documentation/security/policy/language.rst
+++ b/Documentation/security/policy/language.rst
@@ -304,8 +304,9 @@ have ``external:yes`` set as the label.
 Limitations
 ~~~~~~~~~~~
 
-``toServices`` statements cannot be combined with ``toPorts`` statements in the
-same rule.
+``toServices`` statements must not be combined with ``toPorts`` statements in the
+same rule. In the presence of a ``toPorts`` statement, ``toServices`` does nothing
+and as a result all egress traffic to port(s) specified by ``toPorts`` is allowed.
 
 .. _Entities based:
 

--- a/Documentation/security/policy/language.rst
+++ b/Documentation/security/policy/language.rst
@@ -305,8 +305,7 @@ Limitations
 ~~~~~~~~~~~
 
 ``toServices`` statements must not be combined with ``toPorts`` statements in the
-same rule. In the presence of a ``toPorts`` statement, ``toServices`` does nothing
-and as a result all egress traffic to port(s) specified by ``toPorts`` is allowed.
+same rule. If a rule combines both these statements, the policy is rejected.
 
 .. _Entities based:
 

--- a/pkg/policy/api/rule_validation.go
+++ b/pkg/policy/api/rule_validation.go
@@ -190,7 +190,7 @@ func (e *EgressRule) sanitize() error {
 		"ToCIDRSet":   true,
 		"ToEndpoints": true,
 		"ToEntities":  true,
-		"ToServices":  true,
+		"ToServices":  false, // see https://github.com/cilium/cilium/issues/20067
 		"ToFQDNs":     true,
 		"ToGroups":    true,
 	}

--- a/pkg/policy/api/rule_validation_test.go
+++ b/pkg/policy/api/rule_validation_test.go
@@ -627,7 +627,7 @@ func (s *PolicyAPITestSuite) TestToServicesSanitize(c *C) {
 	}
 
 	err := toServicesL3L4.Sanitize()
-	c.Assert(err, IsNil)
+	c.Assert(err, NotNil)
 
 }
 

